src/util/useCheckCanEdit.ts

```diff
@@ -1,19 +1,12 @@
 import useOktaTokens from "../hooks/useOktaTokens";
 import { Acl } from "@madie/madie-models/dist/Measure";
 
-const useCheckUserCanEdit = (
-  createdBy: string,
-  acls: Array<Acl>,
-  draft: boolean = true,
-  editTestsOnVersionedMeasures?: boolean
-): boolean => {
+const useCheckUserCanEdit = (createdBy: string, acls: Array<Acl>): boolean => {
```

Comment:
The test case uses a third argument not in the signature. Should it be defaulted or removed from the test?